tools/twix/src/panels/enum_plot.rs

```diff
@@ -154,8 +155,12 @@ impl SegmentRow {
     }
 
     fn show_settings(&mut self, ui: &mut Ui, nao: Arc<Nao>) {
-        let subscription_field =
-            ui.add(CompletionEdit::readable_paths(&mut self.path, nao.as_ref()));
+        let subscription_field = ui.add(NaoPathCompletionEdit::new(
+            ui.next_auto_id().with(ui.id()).with("enum-plot"),
```

Comment:
![image](https://github.com/user-attachments/assets/8edea4c3-a0c0-4548-97e6-b4bdfc9950eb)
This is the id of the ui without `next_auto_id`. Since the id is the same, the completion edit breaks.

Reply:
But looking this up I found the function `auto_id_with`, which is a shortcut for what I did before. I applied this now